statistician/constraints.txt

```diff
@@ -52,14 +52,14 @@ netCDF4==1.5.7
 networkx==2.6.2
 numexpr==2.7.3
 numpy==1.21.2
-odc-algo==0.2.1.dev2568
```

Comment:
Please be aware that I just merged a PR with all ODC libs updated to the latest versions
Just avoiding conflicts.

odc-aio==0.2.1.dev2476
odc-apps-cloud==0.2.1.dev2622
odc-apps-dc-tools==0.2.1.dev2622
odc-aws==0.2.1.dev2476
odc-index==0.2.1.dev2420
odc-io==0.2.1.dev2622
odc-ppt==0.2.1.dev2400

Reply:
Hi, @carioca-au, thanks a lot for your information. Consider your merge is newer than mine. I believe I don't need this PR anymore.